content/800-guides/330-github-actions.mdx

```diff
@@ -0,0 +1,583 @@
+---
+title: 'How to manage Prisma Postgres with GitHub Actions'
+metaTitle: 'How to manage Prisma Postgres with GitHub Actions'
+description: 'Provision and manage Prisma Postgres databases per pull request using GitHub Actions and Prisma Management API'
+sidebar_label: 'GitHub Actions'
```

Comment:
Perhaps putting this under "Development tools" with something like "Preview dbs for PRs" or similar may make more sense to help people quickly find this